### ISSUE TITLE
Problem list test failing on CI

### ISSUE THREAD

**Issue by BetsyMcPhail** [OSEHRA/VistA#44]
e.g. https://code.osehra.org/CDash/testDetails.php?test=2800897&build=40987

Error log:

```
...............
PXRMOUT^ZZRGUTEX - 150374442,MD+1^ICDEX,%YDB-E-FALLINTOFLST, Fall-through to a label with formallist is not allowed
.............

Ran 1 Routine, 8 Entry Tags
Checked 29 tests, with 0 failures and encountered 1 error.

```

Note: This is not an issue when using Cache.

**Comment by BetsyMcPhail** [comment]
@shabiel  - any ideas what is happening here?

**Comment by shabiel** [comment]
Yes Betsy. There is a missing QUIT. Intersystems tries to be helpful
and put a QUIT for you when you forget it. That's why you don't see it
in Cache. In general, in programming languages, it's considered the
standard practice to tell you if you are missing a semi-colon rather
than assume that you meant to put it there. So we should fix this. I
got the code below; yes, it's missing a quit.

```
MD(FILE,IEN,CDT,ARY,FLAG) ; MDC DRGs
 D MD^ICDEXD2($G(FILE),$G(IEN),$P($G(CDT),".",1),.ARY,$G(FLAG))
EFM(EDT) ; Convert External Date to FM (ICDGTDRG)
```

**Comment by shabiel** [comment]
I forgot to mention: I made a fix for the routine and a unit test
here: https://github.com/shabiel/foia-vista-fixes. I also updated the
submission to the tech journal.

--Sam

**Comment by shabiel** [comment]
Should be fixed by PR #50. I would like you to verify.

**Comment by BetsyMcPhail** [comment]
Thanks Sam - Problem List test is passing on #50.
